README.md

```diff
@@ -154,14 +175,19 @@ How many valid sentences are there in this language?
 
 ## A2.6
 
-«replace this with your answer»
+Adjectives: four options (none, lazy, smelly, lazy and smelly)
```

Comment:
Probably true for your interpreter, but the language itself has an infinite number of sentences

# -3